docs/LibraryEvolution.rst

```diff
@@ -269,15 +269,13 @@ Always Emit Into Client
 A function, computed property or subscript annotated as ``@_alwaysEmitIntoClient``
 is similar to an ``@inlinable`` declaration, except the declaration is
 not part of the module's ABI, meaning that the client must always emit
-their own copy.
-
-As a result, removing a declaration annotated as ``@_alwaysEmitIntoClient``
-is a binary-compatible source-breaking change.
-
-.. admonition:: TODO
-
-    The implementation of ``@_alwaysEmitIntoClient`` is incomplete and
-    should probably graduate to having its own evolution proposal.
+their own copy. As a result:
+- Removing a declaration annotated as ``@_alwaysEmitIntoClient`` is a
```

Comment:
A blank line is required before the first list item, otherwise the [output][] is broken.

[output]: <https://github.com/apple/swift/blob/d04b019cd634db698e4a4bcbe79af2351fa8e90c/docs/LibraryEvolution.rst#always-emit-into-client>

```suggestion
their own copy. As a result:

- Removing a declaration annotated as ``@_alwaysEmitIntoClient`` is a
```